web-bluetooth/battery-level.html

```diff
@@ -0,0 +1,68 @@
+---
+feature_name: Battery Level Web Bluetooth
```

Comment:
I'm not sure the conventions for feature names, but maybe this is the Web Bluetooth feature with the Battery Level sub-feature? "Battery Level « Web Bluetooth" or "Web Bluetooth / Battery Level"?